.circleci/config.yml

```diff
@@ -126,6 +126,60 @@ jobs:
             - ~/.gradle
           key: jars-{{ checksum "android/build.gradle" }}-{{ checksum  "android/app/build.gradle" }}
 
+  # Create signed Android release
+  release-android:
+    <<: *defaults
+
+    docker:
+      - image: circleci/android:api-27-node8-alpha
+
+    environment:
+      - TERM: "dumb"
+      - ANDROID_SDK_BUILD_TOOLS_REVISION: "23.0.1"
+      - ANDROID_SDK_BUILD_API_LEVEL="23": "23"
+
+    steps:
+      # Restore workflow workspace
+      - attach_workspace:
+          at: /home/circleci
+
+      # Restore sdk cache
+      - restore_cache:
+          keys:
+            - v1-android-sdkmanager-packages-{{ checksum "scripts/circleci-android-setup.sh" }}
+            - v1-android-sdkmanager-packages-
+
+      # Install Android SDK
+      - run:
+          command: ./scripts/circleci-android-setup.sh
+
+      # Restore jars cache
+      - restore_cache:
+          key: jars-{{ checksum "android/build.gradle" }}-{{ checksum  "android/app/build.gradle" }}
+
+      # Set Ruby Version for chruby
+      - run: echo "ruby-2.4" > .ruby-version
+
+      # Install bundle dependencies
+      - run:
+          name: Bundle Install
+          command: bundle check || bundle install
+
+      # Fetch CocoaPods specs
+      - run: curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
```

Comment:
is this needed for the Android build?

Reply:
You are right pods not needed, I am going to remove it.